docs/how-to/quantization.md

```diff
@@ -137,31 +87,60 @@ quantized_model = quantize_qat(model_fp32, model_quant)
-After calling the API, three different format of calibration tables are generated with filename calibration.* (FlatBuffers, Python dictionary and plain text).
-Note: In order to include all tensors from the model for better calibration, please run symbolic_shape_infer.py first. (see [here](./../reference/execution-providers/TensorRT-ExecutionProvider.html#sample))
-#### Example
-{: .no_toc }
-```
-data_reader = YoloV3DataReader(calibration_dataset, model_path=augmented_model_path)
-calibrate.collect_data(data_reader)
-calibrate.compute_range()
-```
-Please see [E2E_example_model/e2e_user_yolov3_example.py](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/quantization/E2E_example_model/object_detection/trt/yolov3/e2e_user_yolov3_example.py) for more details.
-### Evaluation for quantization
-#### COCO dataset evaluation
-This tool integrates the COCO dataset API to evaluate model prediction. Please make sure to install COCO API first (```pip install pycocotools```).
-#### Example
-{: .no_toc }
-```
-dr = YoloV3DataReader(validation_dataset, model_path=model_path, start_index=i, size_limit=stride, batch_size=20, is_evaluation=True)
-evaluator = YoloV3Evaluator(model_path, dr, providers=providers)
-evaluator.predict()
-results += evaluator.get_result()
-...
-evaluator.evaluate(results, annotations)
-```
-Please see [E2E_example_model/e2e_user_yolov3_example.py](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/quantization/E2E_example_model/object_detection/trt/yolov3/e2e_user_yolov3_example.py) for more details.
+## Which quantization data type to use?
+Quantization represents value with 8 bit, which can be either int8 and uint8. Combining with activation and weight, the data format can be (activation:uint8, weight:uint8), (activation:uint8, weight:int8), etc.
+
+Let's use U8U8 as as shorthand for (activation:uint8, weight:uint8), and U8S8 for (activation:uint8, weight:int8), and S8U8, S8S8 for other two formats.
+
+Currently, OnnxRuntime CPU only supports activation with type uint8, i.e., U8X8 only.
+
+### x86-64
+#### AVX2
+Try U8U8 firstly, and then U8S8.
+- Performance
+U8S8 leverage VPMADDUBSW instruction but U8U8 kernel can process 6 rows at a time versus 4 rows for the U8S8 kernel, and U8U8 sequence is two instructions vs three instructions for U8S8. Thus, this balance ends up with u8u8 just in reach of U8S8 for older HW (Broadwell)
+- Accuracy
+VPMADDUBSW has saturation issue, thus U8S8 needs to use [reduce_range](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/quantization/quantize.py) if accuracy is not good enough.
+
+#### AVX512
+U8S8 is much faster, but may suffer saturation issue.
+- Performance
+U8S8 can only do twice mul/adds per instruction as much as U8U8. Typically U8S8 sequence ends up winning, can be twice as fast as u8u8.

```

Comment:
"U8S8 can do twice as many mul/adds per instruction compared to U8U8 so can be twice as fast."